dojo/tools/burp_api/parser.py

```diff
@@ -106,14 +106,18 @@ def get_clean_base64(self, value):
         if value is not None:
             for segment in value:
                 if segment["type"] == "DataSegment":
-                    output += base64.b64decode(segment["data"]).decode()
+                    data = base64.b64decode(segment["data"])
+                    try:
+                        output += data.decode()
+                    except UnicodeDecodeError:
+                        output += data.decode('latin1')
```

Comment:
What about: If `decode` would fail, then do:

output += "Decoding failed. Thus, tried to decode with `latin1`. The result is the following one: \n"
output += data.decode('latin1')